### PR TITLE
ROCANA-7883: Fork metrics-jvm-nonaccumulating

### PR DESCRIPTION
**Pull request by joey**
- Set the version of commons-lang3 to 3.0
- Set the version of metrics-jvm-nonaccumulating to
  1.0.2-rocana2.0.0-SNAPSHOT
- Update the gradle wrapper to use the version that supports better ide
  debugging.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 group=com.heapwhisperer.metrics.jvm
-version=1.0.2
+version=1.0.2-rocana2.0.0-SNAPSHOT
```
Comment (bradcupit) [inline gradle.properties:2]: Bashar [said](https://github.com/gburton1/metrics-jvm-nonaccumulating/pull/1#discussion_r85027764):

> Does this mean we have to make a release build of this dependency before we release 2.0.0?

Joey replied:

> Yup, that's the pattern we take with java library forks like Kite.

I really like the idea of being consistent in how we do things.

On the other hand, it's not bad to reflect on how we do things and make sure it's the most efficient.

Recently we've revisited this process with the Impala fork and are having second thoughts.

Comment (joey) [reply gradle.properties:2]: We haven't been releasing Java libraries with every release. I could mark it as `rocana2` to indicate that it's tied to the 2.x.x line of Rocana?

Comment (bradcupit) [reply gradle.properties:2]: What about just `rocana1`, then if we issue another build we make it `rocana2`?
Or if that's not granular enough: `rocana-1.0.0`, meaning, the 1.0.0 release of this lib?

p.s. I'm not going to let this hold up the PR, I just wanted to start the discussion

**Comment by bradcupit** [comment]
+1
The `rocana2.0.0-SNAPSHOT` fits with our current style, and though I like the idea of discussing that, that's tangential to this PR.

**Pull request merged**